transifex/pull.py

```diff
@@ -317,8 +83,7 @@ def parse_arguments():
     args = parse_arguments()
 
     if args.clone_url:
-        repo = Repo(args.clone_url, skip_compilemessages=args.skip_compilemessages)
-        pull(repo)
+        pull(args.clone_url, skip_compilemessages=args.skip_compilemessages)
     else:
         logger.info('No arguments provided. Using settings.yaml.')
```

Comment:
@amangano-edx go ahead and remove it. When the script was initially developed it was run like this, but now we use Jenkins jobs to invoke the script for a specific repo.